src/lib/chips/chips.scss

```diff
@@ -16,8 +16,10 @@ $mat-chips-chip-margin: 8px;
 }
 
 .mat-chip:not(.mat-basic-chip) {
+  @include mat-elevation-transition;
   display: inline-flex;
-  padding: $mat-chip-vertical-padding $mat-chip-horizontal-padding $mat-chip-vertical-padding $mat-chip-horizontal-padding;
+  padding: $mat-chip-vertical-padding $mat-chip-horizontal-padding $mat-chip-vertical-padding
```

Comment:
here can use `padding: $mat-chip-vertical-padding $mat-chip-horizontal-padding` instead of 4 values?